core/src/main/java/hudson/ProxyConfiguration.java

```diff
@@ -376,7 +413,7 @@ public FormValidation doCheckPort(@QueryParameter String value) {
         @RequirePOST
```

Comment:
```suggestion
        @RequirePOST
        @Restricted(NoExternalUse.class)
```

Reply:
i do not see external usages of this method, so it looks OK to change the parameter type here